sycl/include/CL/sycl/property_list.hpp

```diff
@@ -39,6 +39,8 @@ class context_bound;
 
 namespace queue {
 class enable_profiling;
+class in_order;
+class out_of_order;
```

Comment:
```suggestion

```

Reply:
@jbrodman, I think you forgot to remove this unused forward declaration.

Reply:
Good catch.

Reply:
Fixed.